Snabble/UI/DynamicStack/WidgetButtonView.swift

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
Der Button wird nicht in der vollen breite dargestellt, wie es aktuell gefordert ist.